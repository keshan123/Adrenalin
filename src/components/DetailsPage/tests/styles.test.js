import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import {
  PageContainer,
  DetailsContainer,
  DetailsImage,
  DetailsInfo,
  DetailsTitle,
  DetailsQuestionsContainer,
  DetailsQuestionsTitle,
  DetailsQuestionsBody,
  DetailsTagContainer,
  DetailsImageTag,
} from '../styles';

test('PageContainer renders', () => {
  const tree = renderer.create(
      <PageContainer/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('DetailsContainer renders', () => {
  const tree = renderer.create(
      <DetailsContainer/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('DetailsImage renders', () => {
  const tree = renderer.create(
      <DetailsImage/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('DetailsInfo renders', () => {
  const tree = renderer.create(
      <DetailsInfo/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('DetailsTitle renders', () => {
  const tree = renderer.create(
      <DetailsTitle/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('DetailsQuestionsContainer renders', () => {
  const tree = renderer.create(
      <DetailsQuestionsContainer/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('DetailsQuestionsTitle renders', () => {
  const tree = renderer.create(
      <DetailsQuestionsTitle/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('DetailsQuestionsBody renders', () => {
  const tree = renderer.create(
      <DetailsQuestionsBody/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('DetailsTagContainer renders', () => {
  const tree = renderer.create(
      <DetailsTagContainer/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('DetailsImageTag renders', () => {
  const tree = renderer.create(
      <DetailsImageTag/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
