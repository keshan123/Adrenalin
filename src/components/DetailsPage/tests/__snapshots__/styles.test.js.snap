// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailsContainer renders 1`] = `
.c0 {
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 0 93px;
}

@media (max-width:830px) {
  .c0 {
    padding: 0;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

<div
  className="c0"
/>
`;

exports[`DetailsImage renders 1`] = `
.c0 {
  background-image: url();
  background-size: cover;
  height: 1107px;
  margin-top: 62px;
  width: 650px;
}

<div
  className="c0"
/>
`;

exports[`DetailsImageTag renders 1`] = `
.c0 {
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  background-color: #F5F5F5;
  box-shadow: 0px 4px 4px -2px rgba(0,0,0,0.3);
  color: rgba(0,0,0,0.75);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 12px;
  height: 24px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  left: 0;
  margin-left: -37px;
  margin-top: 99px;
  position: absolute;
  top: 0;
  -webkit-transform: rotate(270deg);
  -ms-transform: rotate(270deg);
  transform: rotate(270deg);
  width: 98px;
}

<div
  className="c0"
/>
`;

exports[`DetailsInfo renders 1`] = `
.c0 {
  height: 800px;
  margin-left: 80px;
  margin-top: 123px;
  overflow: scroll;
  width: 564px;
}

@media (max-width:830px) {
  .c0 {
    margin-left: 0;
  }
}

<div
  className="c0"
/>
`;

exports[`DetailsQuestionsBody renders 1`] = `
.c0 {
  color: #333333;
  font-family: Georgia,Times,Times New Roman,serif;
  font-family: Georgia;
  font-size: 21px;
  font-weight: normal;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
  line-height: 37px;
  margin-top: 10px;
}

<div
  className="c0"
/>
`;

exports[`DetailsQuestionsContainer renders 1`] = `
.c0 {
  -webkit-align-text: left;
  -ms-flex-line-packalign-text: left;
  align-text: left;
  margin-bottom: 30px;
}

<div
  className="c0"
/>
`;

exports[`DetailsQuestionsTitle renders 1`] = `
.c0 {
  font-size: 32px;
  font-weight: bold;
}

<div
  className="c0"
/>
`;

exports[`DetailsTagContainer renders 1`] = `
.c0 {
  position: relative;
}

<div
  className="c0"
/>
`;

exports[`DetailsTitle renders 1`] = `
.c0 {
  -webkit-align-text: left;
  -ms-flex-line-packalign-text: left;
  align-text: left;
  font-family: Arial,Helvetica Neue,Helvetica,sans-serif;
  font-size: 68px;
  font-weight: bold;
  margin-bottom: 65px;
}

<div
  className="c0"
/>
`;

exports[`PageContainer renders 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  className="c0"
/>
`;
